keyboards/kumo/keymaps/SeaSpider/rules.mk

```diff
@@ -0,0 +1,70 @@
+#     processor frequency in Hz. You can then use this symbol in your source code to
+#     calculate timings. Do NOT tack on a 'UL' at the end, this will be done
+#     automatically to create a 32-bit value in your source code.
+#
+#     This will be an integer division of F_USB below, as it is sourced by
+#     F_USB after it has run through any CPU prescalers. Note that this value
+#     does not *change* the processor frequency - it should merely be updated to
+#     reflect the processor speed set externally so that the code can use accurate
+#     software delays.
+F_CPU = 16000000
+
+# LUFA specific
+#
+# Target architecture (see library "Board Types" documentation).
+ARCH = AVR8
+
+# Input clock frequency.
+#     This will define a symbol, F_USB, in all source code files equal to the
+#     input clock frequency (before any prescaling is performed) in Hz. This value may
+#     differ from F_CPU if prescaling is used on the latter, and is required as the
+#     raw input clock is fed directly to the PLL sections of the AVR for high speed
+#     clock generation for the USB and other AVR subsections. Do NOT tack on a 'UL'
+#     at the end, this will be done automatically to create a 32-bit value in your
+#     source code.
+#
+#     If no clock division is performed on the input clock inside the AVR (via the
+#     CPU clock adjust registers or the clock division fuses), this will be equal to F_CPU.
+F_USB = $(F_CPU)
+
+# Interrupt driven control endpoint task(+60)
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+
+# Boot Section Size in *bytes*
+#   Teensy halfKay   512
+#   Teensy++ halfKay 1024
+#   Atmel DFU loader 4096
+#   LUFA bootloader  4096
+#   USBaspLoader     2048
+OPT_DEFS += -DBOOTLOADER_SIZE=4096
+#BOOTLOADER= qmk-dfu
+
+# was
+# OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+# OPT_DEFS += -DBOOTLOADER_SIZE=4096
+
+# Build Options
+#   comment out to disable the options.
+#
+CONSOLE_ENABLE    = no # Console for debug(+400)
+COMMAND_ENABLE    = no  # Commands for debug and configuration
+UNICODE_ENABLE    = no  # added on 08-18-2018 12:21 PM CST
+COMMAND_ENABLE    = no  # required for space cadet, when shift is also (.
+MIDI_ENABLE       = no  # MIDI controls
+AUDIO_ENABLE      = no  # Audio
+UNICODE_ENABLE    = no  # Unicode
+BLUETOOTH_ENABLE  = no  # Enable Bluetooth with the Adafruit EZ-Key HID
+SLEEP_LED_ENABLE  = no
+BOOTMAGIC_ENABLE  = yes # Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE   = yes # Mouse keys(+4700)
+EXTRAKEY_ENABLE   = yes	# Audio cones	# Audio control and System control(+450)
+NKRO_ENABLE       = yes # USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+RGBLIGHT_ENABLE   = yes # Enable keyboard underlight functionality (+4870)
+BACKLIGHT_ENABLE  = yes # Enable keyboard backlight functionality (+1150)
+TAP_DANCE_ENABLE  = yes #
+AUTO_SHIFT_ENABLE = yes  # Enables autoshift
```

Comment:
It appears that this is a `rules.mk` file for a specific keymap. You actually don't need most of this stuff in here, only the lines that are different from the keyboard's base `rules.mk` file (`keyboards/kumo/rules.mk`).

Actually, it looks like you got the base and keymap `rules.mk` files reversed. _This_ should be the core `keyboards/kumo/rules.mk` file since it has all the `MCU`, `ARCH` etc. definitions; `keyboards/kumo/keymaps/SeaSpider/rules.mk should only contain the few things that are different, namely:
```makefile
CONSOLE_ENABLE = no
COMMAND_ENABLE = no
```
(by the way, I don't recommend disabling Command)

Similarly, only the main `config.h` file (`keyboards/kumo/config.h`) should contain the `VENDOR_ID`, `PRODUCT_ID`, etc.; the keymap `config.h` don't need any of this.